content/efcore/getting-started/loading-related-data.md

```diff
@@ -1,4 +1,6 @@
-# Loading Related Data  
+# Loading Related Data
+
+In this lesson, we will cover how to load non-primitive types from the database, how to chain commands to include multiple layers of related data, and how to use explicit loading and aggregate functions.
```

Comment:
Chaining commands to include multiple layers of related data won't really mean anything to them. Could we change this to:

"In this lesson, we will learn how to load non-primitive types from the database using both eager loading and explicit loading, as well as how to use aggregate functions to perform calculations on our data without loading it from the database."